pyuvdata/uvdata/uvh5.py

```diff
@@ -1214,6 +1294,17 @@ def _write_header(self, header):
             header["eq_coeffs_convention"] = np.string_(self.eq_coeffs_convention)
         if self.flex_spw_id_array is not None:
             header["flex_spw_id_array"] = self.flex_spw_id_array
+        if self.phase_center_id_array is not None:
+            header["phase_center_id_array"] = self.phase_center_id_array
+        if self.Nphase is not None:
+            header["Nphase"] = self.Nphase
+
+        # Write out the catalog, if available
+        if self.phase_center_catalog:
+            phase_dict = header.create_group("phase_center_catalog")
+            for k in self.phase_center_catalog.keys():
+                # Dictionary entries can be written out as JSON-formatted strings.
+                phase_dict[k] = json.dumps(self.phase_center_catalog[k])
```

Comment:
Got it, thanks. Unfortunately, I think we still need to wrap the `json.dumps` in a call to `np.string_`, just to make sure it's being saved as fixed-length ASCII strings (which is best for interoperability with other languages).

Reply:
Fair enough -- thusly wrapped in a1c62ff.